plugins/home/src/components/HeaderWorldClock/HeaderWorldClock.tsx

```diff
@@ -69,23 +72,26 @@ function getTimes(clockConfigs: ClockConfig[]) {
 }
 
 /** @public */
-export const HeaderWorldClock = (props: { clockConfigs: ClockConfig[] }) => {
-  const { clockConfigs } = props;
+export const HeaderWorldClock = (props: {
```

Comment:
@awanlin  and @emmaindal is it a good idea to also add this component to our [storybook](https://backstage.io/storybook/?path=/story/plugins-home-templates--default-template)? Or even improve the API reporting by adding an  [example](https://api-extractor.com/pages/tsdoc/tag_example/) of how to customize the time format?

Reply:
Good idea with storybook @camilaibs, feel free to add @awanlin if you want otherwise we can do it separately, I think we have a couple of other home page components that can be added there too! 

Love the API report suggestions though!

Reply:
Yes! Ditto what @emmaindal said. Hold off on the merge and I'll add these by the end of the week

Reply:
This ended up a bit bigger change as I moved this component to the same location as the other, felt a bit lonely where it was. I've added a story with two examples - Default and 24hr Clocks. I also added an `@example` using the API Report Syntax but it's not showing up in the API Report :(